parts/appendices/B.fodt

```diff
@@ -13581,10 +13854,229 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
        <text:list-item>
         <text:list>
          <text:list-item text:start-value="1">
-          <text:p text:style-name="P1298">Item1</text:p>
+          <text:p text:style-name="P1298">Avoid sending and receiving from the same Buffer (<text:a xlink:type="simple" xlink:href="https://github.com/OPM/opm-simulators/pull/5297" text:style-name="Internet_20_link" text:visited-style-name="Visited_20_Internet_20_Link">#5297</text:a>). </text:p>
          </text:list-item>
          <text:list-item>
-          <text:p text:style-name="P1298"/>
+          <text:p text:style-name="P1298">Redistribute gaslift for groups with an ALQ limit (<text:a xlink:type="simple" xlink:href="https://github.com/OPM/opm-simulators/pull/5296" text:style-name="Internet_20_link" text:visited-style-name="Visited_20_Internet_20_Link">#5296</text:a>). Previousy, gaslift was only redistributed for groups limited by rate. </text:p>
+         </text:list-item>
+         <text:list-item>
+          <text:p text:style-name="P1298">Fixed output of CO2 mass in place (<text:a xlink:type="simple" xlink:href="https://github.com/OPM/opm-simulators/pull/5324" text:style-name="Internet_20_link" text:visited-style-name="Visited_20_Internet_20_Link">#5324</text:a>). </text:p>
+         </text:list-item>
+         <text:list-item>
+          <text:p text:style-name="P1298">Initialise the reservoir temperature in CO2STORE restart runs using the initial reservoir temperature if no temperatures are stored in the restart file, as is now the case for isothermal runs (<text:a xlink:type="simple" xlink:href="https://github.com/OPM/opm-simulators/pull/5331" text:style-name="Internet_20_link" text:visited-style-name="Visited_20_Internet_20_Link">#5331</text:a>). </text:p>
+         </text:list-item>
+         <text:list-item>
+          <text:p text:style-name="P1298">Mark the option 'BASIC=0' as allowed for the RPTRST keyword (<text:a xlink:type="simple" xlink:href="https://github.com/OPM/opm-simulators/pull/5335" text:style-name="Internet_20_link" text:visited-style-name="Visited_20_Internet_20_Link">#5335</text:a>). </text:p>
+         </text:list-item>
+         <text:list-item>
+          <text:p text:style-name="P1298">Corrected definitions of GMDS and GMGP summary vectors (<text:a xlink:type="simple" xlink:href="https://github.com/OPM/opm-simulators/pull/5292" text:style-name="Internet_20_link" text:visited-style-name="Visited_20_Internet_20_Link">#5292</text:a>). </text:p>
+         </text:list-item>
+         <text:list-item>
+          <text:p text:style-name="P1298">Fixed a typo that would cause problems when compiling using CUD-aware MPI (<text:a xlink:type="simple" xlink:href="https://github.com/OPM/opm-simulators/pull/5368" text:style-name="Internet_20_link" text:visited-style-name="Visited_20_Internet_20_Link">#5368</text:a>).  </text:p>
```

Comment:
```suggestion
          <text:p text:style-name="P1298">Fixed a typo that would cause problems when compiling using CUDA-aware MPI (<text:a xlink:type="simple" xlink:href="https://github.com/OPM/opm-simulators/pull/5368" text:style-name="Internet_20_link" text:visited-style-name="Visited_20_Internet_20_Link">#5368</text:a>).  </text:p>
```